server/apps/archive/archive_crop.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import superdesk
+from flask import request
+from superdesk import get_resource_service
+from superdesk.resource import Resource
+from .common import item_url
+from superdesk.services import BaseService
+from superdesk.errors import SuperdeskApiError
+from superdesk.media.media_operations import crop_image, process_file_from_stream
+from superdesk.upload import UploadService, url_for_media
+
+
+class ArchiveCropResource(Resource):
+    endpoint_name = 'archive_crop'
+    url = 'archive/<{0}:item_id>/crop/<{0}:crop_name>'.format(item_url)
+    schema = {
+        'CropLeft': {'type': 'integer'},
```

Comment:
It has been done like this before. Again you need either consistency or much bigger change to fix the previously done stuff.